src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -45,16 +35,22 @@ import { LoggingService } from "../../logging/logging.service";
     MatTooltipModule,
     MatInputModule,
     MatCheckboxModule,
+    AsyncPipe,
+    ErrorHintComponent,
+    BasicAutocompleteComponent,
+    MatSlideToggle,
+    FormsModule,
   ],
   standalone: true,
 })
 @UntilDestroy()
-export class EntitySelectComponent<E extends Entity> implements OnChanges {
-  readonly separatorKeysCodes: number[] = [ENTER, COMMA];
+export class EntitySelectComponent<
+  E extends Entity,
+  T extends string[] | string = string[],
```

Comment:
This is a bit annoying for typing to make things work as a single and multi select, although the code and template works very well for both ...
If you have any ideas for improvement here, @tomwwinter ,  happy to hear them!

Reply:
Is there a way to pass always an array?

Reply:
This component is used in `EditEntityComponent`, which in turn already receives a `FormControl` from parent through which the value input and output is handled (and which maps to either a single (non-array) entity property or a multi entity-array property). We could maybe use value change subscriptions to transform the value a level higher or create a internal formControl and connect that ... but because the formControl is created a few levels higher up and also contains details like validators, I am a bit wary of the side effects of that.